### PR TITLE
Updated ghost and node

### PR DESCRIPTION
**Pull request by blaze33**
Sorry for the new pull request but I broke the other one (#106) by committing  my personal fork of casper without thinking that it would update the pull request. This one is from a specific branch so no problem this time.

### CODE REVIEW

**Comment by cobyism** [comment]
No worries, @blaze33—thanks for sorting it out and opening another PR!

File: package.json [hunk 1]
```diff
@@ -8,16 +8,16 @@
...
-    "ghost": "0.11.3",
+    "casper": "TryGhost/Casper#1.3.6",
+    "ghost": "0.11.8",
     "ghost-s3-storage-adapter": "3.0.4",
     "ncp": "^2.0.0",
     "pg": "latest"
   },
   "engines": {
-    "node": "~0.12.0"
+    "node": "^6.10.3"
```
Comment (cobyism) [inline package.json]: What are your thoughts on simply using the version string straight from [Ghost’s package.json](https://github.com/TryGhost/Ghost/blob/0.11.8/package.json#L26) (i.e. going with `^4.2.0 || ^6.9.0`)?

Comment (blaze33) [inline package.json]: Hmm, as I noted in the other PR, it doesn't change anything as Heroku chooses node v6.10.3 in both cases, but yeah just to be safe I'll revert to the Ghost version, it'll we clearer for future updates.

Comment (cobyism) [inline package.json]: Yeah, I think it helps to establish more patterns for future decisions regarding this repo too. The more we can defer to outside influences (i.e. upstream ghost) for decisions, the simpler it is for anyone to update things like this going forward. 👍

**Comment by cobyism** [comment]
Awesome stuff. Thanks @blaze33!

**Pull request merged**